cerner/beadledom
# Update Travis configuration + release documentation

What was changed? Why is this necessary?
----------------------------------------

* Updated the release documentation
* Updated the travis configuration & scripts to build and deploy the tagged artifacts to maven central.
* Updated the site publish script to build and update the gh-pages branches automatically everytime the project is released (tagged).
 
How was it tested?
----
* Configured the travis temporarily build my branch.

How to test
----

Update the travis configuration to build a specific branch instead of master and push a commit. Travis should automatically start a build.

Reviewers
----

- [ ] [John Leacox](https://github.com/johnlcox)
- [ ] [Nimesh Subramanian](https://github.com/nimeshsubramanian)
- [ ] [Supriya Lal](https://github.com/lal-s)
- [ ] [Brian van de Boogaard](https://github.com/b-boogaard)
